### ISSUE TITLE
Figure 3 Error

### ISSUE THREAD

**Issue by Algomancer** [abhinavsagar/gvae#1]
Hi, Just a note - the photos you list in figure 3 are all from the dataset.

**Comment by madebyollin** [comment]
Yes, "Figure 3" here is just a crop of Figure 10 from the [Progressive GAN paper](https://arxiv.org/pdf/1710.10196.pdf), which visualizes nearest neighbors from training set.

<img width="917" alt="Screen Shot 2020-08-25 at 04 10 41" src="https://user-images.githubusercontent.com/8733615/91167558-ff37a900-e688-11ea-985e-5969d9daa0c4.png">

Likewise,  "Figure 4" is a crop of Figure 12 from the Progressive GAN paper:
<img width="1010" alt="Screen Shot 2020-08-25 at 04 15 26" src="https://user-images.githubusercontent.com/8733615/91167943-9d2b7380-e689-11ea-88e3-4da22ac917dd.png">

And "Figure 5" is a crop from Figure 7:

<img width="1014" alt="Screen Shot 2020-08-25 at 04 16 19" src="https://user-images.githubusercontent.com/8733615/91168024-c21fe680-e689-11ea-9d87-5a67dafcf013.png">

So, the captions in the GVAE paper are, I think, _technically_ correct in 2/3 cases
> Figure 3: 1024 × 1024 images generated using the CELEBA-HQ dataset.
(Not true; `s/generated using/sampled from`)
> Figure 4: 256 × 256 images generated using LSUN BEDROOM dataset
(Correct, by ProgressiveGAN)
> Figure 5: Sample 256×256 images generated from different LSUN categories
(Correct, by ProgressiveGAN)

**Comment by usamahjundia** [comment]
damn. No wonder the paper makes little if not no sense at all to me. Did i really wasted a whole morning trying to understand the paper and code i wonder
